src/gui/dialog/LatexDialog.cpp

```diff
@@ -97,6 +98,14 @@ void LatexDialog::load()
 {
 	XOJ_CHECK_TYPE(LatexDialog);
 	gtk_text_buffer_set_text(this->textBuffer, this->theLatex.c_str(), -1);
+
+	// preselect all text in the box if desired
+	if (this->preselect) {
+		GtkTextIter start, end;
+		gtk_text_buffer_get_start_iter(this->textBuffer, &start);
+		gtk_text_buffer_get_end_iter(this->textBuffer, &end);
+		gtk_text_buffer_select_range(this->textBuffer, &start, &end);
```

Comment:
Everything except these four lines can be removed from the commit.

Reply:
You're missing something there. We only want to select the default x^2 and don't preselect anything when an existing formula is edited

Reply:
Ahh good call I haven't thought about that 😅